docs/conf.py

```diff
@@ -12,10 +12,11 @@
 import os
 import shutil
 import sys
+from pathlib import Path
+from typing import Dict
 
-__location__ = os.path.join(
-    os.getcwd(), os.path.dirname(inspect.getfile(inspect.currentframe()))
-)
+dir_name = inspect.getfile(inspect.currentframe())  # type: ignore
+__location__ = os.path.join(os.getcwd(), os.path.dirname(dir_name))
```

Comment:
MAYBE: Use ```Path``` than ```join```?

Reply:
This is a weird one where the code works but the types don't match and I didn't want to disturb it but just make sure that the mypy checks were safely ignored. I can only assume that this is used to get access to the file that imports this file during run time so i figured without a good way to test it, I would leave it alone. Happy to upgrade to pathlib across the board, although I wonder why this isn't already a warning in mypy.